papers/Remy2021/sample_cosmos_jz.job

```diff
@@ -24,9 +25,8 @@ module load tensorflow-gpu/py3/2.5.0
 
 for i in {1..6}
 do
-  srun python ./sample_ODE.py --convergence=../data/ktng/ktng_kappa360v2.fits\
+  srun python ./sample_maps.py --convergence=../data/ktng/ktng_kappa360v2.fits\
                         --mask=../data/COSMOS/cosmos_full_mask_0.29arcmin360copy.fits\
-                        --num_steps=10\
                         --sigma_gamma=0.148\
```

Comment:
what do we need sigma_gamma for?

Reply:
it is useless with the ODE sampler indeed